modules/gapi/test/render/gapi_render_tests_ocv.cpp

```diff
@@ -418,6 +449,24 @@ INSTANTIATE_TEST_CASE_P(RenderNV12OCVTestTextsImpl, RenderNV12OCVTestTexts,
                                 Values(LINE_8),
                                 Values(false)));
 
+# ifdef HAVE_FREETYPE
+INSTANTIATE_TEST_CASE_P(RenderBGROCVTestFTextsImpl, RenderBGROCVTestFTexts,
+                        Combine(Values(cv::Size(1280, 720)),
+                            Values(L"你好，世界",
+                                   L"〤〥〦〧〨々〆"),
```

Comment:
Be careful with UTF-8 string literals in C++ code (not all compilers works well).
Consider passing "wchar_t[]" numeric array instead (also avoid comments with non-ASCII chars).

Reply:
Fixed